Practice/assets/javascript/smoothscroll.js

```diff
@@ -0,0 +1,46 @@
+/* 
+The code to activate smooth scroll.
+https://github.com/cferdinandi/smooth-scroll
+
+*/
+var scroll = new SmoothScroll('a[href*="#"]', {
+  speed: 1000,
+  //   speedAsDuration: true
+});
+
+/*
+Using JavaScript to access DOM elements
+If you try to utilize this code, you will need to ensure that the DOM elements exist with the id's or class names used below. 
+*/
+
+const button = document.getElementById("submit");
+const searchBar = document.getElementById("searchBar");
+
+/*
+Add an event listener to the button
+*/
+button.addEventListener("click", function (event) {
+  event.preventDefault();
+  console.log(searchBar.value);
+  alert(searchBar.value);
+});
+
+/*
+Adding event listeners to the page such that when he page scrolls the navigation bar will change.
+
+*/
+const navBar = document.getElementsByClassName("navbar")[0];
+
+console.log(document.getElementsByClassName("navbar"));
+

```

Comment:
Great implementation of this JavaScript logic to ensure that the navigation bar changes on scoll!
I like how you're using a class to add the css and not just accessing the object. 

Nice job